src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentMergeUtils.java

```diff
@@ -1,6 +1,7 @@
 package org.broadinstitute.hellbender.tools.exome;
```

Comment:
Note to self and anyone else who may be reading these comments in a linear fashion: I'm skipping this for now in favor of lighter code; will return later.